doc/sphinx/language/core/conversion.rst

```diff
@@ -85,6 +85,8 @@ to other changes to the proof state.)
 Two terms are :gdef:`α-convertible <alpha-convertible>` if they are syntactically
 equal ignoring differences in the names of variables bound within the expression.
 For example `forall x, x + 0 = x` is α-convertible with `forall y, y + 0 = y`.
+(Internally, Coq represents these two terms identically using de Bruijn indices,
+so α-conversion is not necessary.)
```

Comment:
"not necessary" is not the right phrasing, I'd rather same something like "by construction" or "built-in".

Reply:
Also, is it really "not necessary"? Surely the conversion to de Bruijn indices can be thought of as alpha conversion, hence alpha conversion is always done?

Reply:
How about this?

```
(Internally, Coq represents these two terms using de Bruijn indices,
so explicit α-conversion is not necessary.)
```